src/common/args.cpp

```diff
@@ -617,6 +617,9 @@ std::string ArgsManager::GetHelpMessage() const
             case OptionsCategory::RPC:
                 usage += HelpMessageGroup("RPC server options:");
                 break;
+            case OptionsCategory::IPC:
```

Comment:
I think this should be added to the `rpc_help.py` tests.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30509#discussion_r1733464897

> I think this should be added to the `rpc_help.py` tests.

Is there mistake in this suggestion? This shouldn't be related to RPC

Reply:
Huh, no idea what I was thinking here, but `rpc_help.py` is obviously wrong. Maybe I copy-pasted a wrong value? I think where it could go is the `system.cpp` fuzz tests.